<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>539</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ACM</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>911</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/logo.png);</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>901</width>
      <height>391</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Players">
     <attribute name="title">
      <string>Players</string>
     </attribute>
     <widget class="QPushButton" name="playeAdButton">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>40</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add </string>
      </property>
     </widget>
     <widget class="QPushButton" name="PlayerEditButton">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>80</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Players</string>
      </property>
     </widget>
     <widget class="QTableView" name="playerTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>771</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="Refine">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>290</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Clear">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>330</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quick Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="quickSearch">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="mockInsert">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>250</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Mock Insert</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Competition">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Competition</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Competitions</string>
      </property>
     </widget>
     <widget class="QPushButton" name="compEdit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>80</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QTableView" name="compTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>771</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLineEdit" name="quickSearchComp">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="compAdd">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>40</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quick Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="compViewDraw">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>120</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>View Draw</string>
      </property>
     </widget>
     <widget class="QPushButton" name="compClear">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>330</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Venues">
     <attribute name="title">
      <string>Venues</string>
     </attribute>
     <widget class="QLineEdit" name="quickSearchVenue">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="venueClear">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>330</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="venueEdit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>80</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Venues</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quick Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="venueAdd">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>40</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add </string>
      </property>
     </widget>
     <widget class="QTableView" name="venueTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>771</width>
        <height>321</height>
       </rect>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="Teams">
     <property name="accessibleName">
      <string/>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <attribute name="title">
      <string>Teams</string>
     </attribute>
     <widget class="QTableView" name="teamTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>771</width>
        <height>321</height>
       </rect>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Teams</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quick Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="teamClear">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>330</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="teamEdit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>80</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="quickSearchTeam">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="teamAdd">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>40</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Payments">
     <attribute name="title">
      <string>Payments</string>
     </attribute>
     <widget class="QPushButton" name="paymentClear">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>330</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="quickSearchPayments">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Quick Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Transactions</string>
      </property>
     </widget>
     <widget class="QTableView" name="paymentTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>771</width>
        <height>321</height>
       </rect>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="paymentAdd">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>40</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add </string>
      </property>
     </widget>
     <widget class="QPushButton" name="paymentEdit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>80</y>
        <width>101</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSetting"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSetting">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
